.storybook/main.js

```diff
@@ -1,29 +1,35 @@
-const path = require('path');
-
-module.exports = {
+/** @type { import('@storybook/react-webpack5').StorybookConfig } */
+const config = {
   stories: ['../client/**/*.stories.(jsx|mdx)'],
   addons: [
-    '@storybook/addon-actions',
-    '@storybook/addon-docs',
-    '@storybook/addon-knobs',
```

Comment:
I noticed that you removed some packages from the `addons` array here but kept them in the `package.json`.  Do they need to be kept as dependencies or should they be removed?

Reply:
They should be removed now. They've been deprecated https://www.npmjs.com/package/@storybook/addon-knobs

Actions are built in by default. And knobs have been replaced with `args` and `argTypes` where you can configure controls to play with. I've updated Buttons and Icons stories to use these rather than the knobs that were there.